scripts/contractABI.js

```diff
@@ -39,14 +42,15 @@ function convertContract(name, json, prefs) {
 	return `${prefs._pub ? "pub " : ""}struct ${name} {
 	contract: ethabi::Contract,
 	address: util::Address,
-	do_call: Box<Fn(util::Address, Vec<u8>) -> Result<Vec<u8>, String> + Send ${prefs._sync ? "+ Sync " : ""}+ 'static>,
+	${prefs._explicit_do_call ? "" : `do_call: Box<Fn(util::Address, Vec<u8>) -> Result<Vec<u8>, String> + Send${prefs._sync ? " + Sync " : ""}+ 'static>,`}
```

Comment:
The idea of _explicit_do_call is to pass `do_call` to contract' methods, instead of constructor. That's because Miner() don't have Arc<Client>, but &Client is passed to its methods => I can construct closure when calling a contract' method, but can't make a persistent closure.